<ui version="4.0" >
 <class>GuiPrinterManualSetupBase</class>
 <widget class="QWidget" name="GuiPrinterManualSetupBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PrinterManualSetup</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label_manualSetup" >
     <property name="text" >
      <string>&lt;h1>Manual Printer Setup&lt;/h1></string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="text" >
      <string>Please select a manufacturer from the manufacturer list.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="labelManufacturers" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>200</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text" >
          <string>&amp;Manufacturers</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetManufacturers" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="labelModels" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>M&amp;odels</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetModels" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="labelDrivers" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Drivers</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetDrivers" />
       </item>
       <item>
        <widget class="QPushButton" name="buttonDriverDetails" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Sho&amp;w Driver Details</string>
         </property>
         <property name="shortcut" >
          <string>Alt+W</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="1" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="labelPorts" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Printer Port</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>comboBoxPorts</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPorts" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>New Item</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="labelPrintername" >
         <property name="text" >
          <string>Printername</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>editPrinterName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPrinterName" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>MyMoneypennyPrinter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="text" >
        <string>Bac&amp;k</string>
       </property>
       <property name="shortcut" >
        <string>Alt+K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHelp" >
       <property name="text" >
        <string>He&amp;lp</string>
       </property>
       <property name="shortcut" >
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>331</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Sa&amp;ve Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>editPrinterName</tabstop>
  <tabstop>comboBoxPorts</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonBack</tabstop>
  <tabstop>buttonHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
